### PR TITLE
Reverse matrix

### PR DESCRIPTION
**Pull request by toor1245**

### CODE REVIEW

File: tests/MatrixDotNet.PerformanceTesting/Matrix/BenchReverse.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+{
+    [MemoryDiagnoser]
+    public class BenchReverse : PerformanceTest
+    {
+        private Matrix<int> _matrix;
+        private Matrix<byte> _matrixByte;
+        private Matrix<float> _matrixFloat;
+        private Matrix<double> _matrixDouble;
+        
+        [GlobalSetup]
```
Comment (Dilorfin) [inline tests/MatrixDotNet.PerformanceTesting/Matrix/BenchReverse.cs:16]: May be here should be not Global but Iteration Setup?

File: tests/MatrixDotNet.PerformanceTesting/Other/BenchCount.cs [hunk 2]
```diff
@@ -5,7 +5,7 @@
 namespace MatrixDotNet.PerformanceTesting.Other
 {
     [RyuJitX64Job]
-    public class CountBench : PerformanceTest
+    public class BenchCountBench : PerformanceTest
```
Comment (Dilorfin) [inline tests/MatrixDotNet.PerformanceTesting/Other/BenchCount.cs:8]: After several name changes. I think that it should be different prs next time. One for "Reverse matrix" and one for "Perf tests names change".

File: tests/MatrixDotNet.UnitTesting/VectorTests/VectorBasicTest.cs [hunk 3]
```diff
@@ -1,4 +1,7 @@
-﻿using MatrixDotNet.Exceptions;
+﻿using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using MatrixDotNet.Exceptions;
+using MatrixDotNet.Extensions.Performance;
```
Comment (Dilorfin) [inline tests/MatrixDotNet.UnitTesting/VectorTests/VectorBasicTest.cs:4]: For what changes do this namespaces are needed here?

**Review by Dilorfin — APPROVED** [review]

File: tests/MatrixDotNet.PerformanceTesting/Matrix/BenchReverse.cs (hunk 1 above)
Comment (toor1245) [reply tests/MatrixDotNet.PerformanceTesting/Matrix/BenchReverse.cs:16]: In this case, I think need to use  GlobalSetup this data need only initialize we compare reverse without initializing on each step

**Review by Dilorfin — APPROVED** [review]
